packages/components/src/css/mixin/color.scss

```
@@ -0,0 +1,27 @@
+// Quoted: bare `gray` would resolve to the CSS named color, not to a name we can slice.
+$colors: 'gray', 'primary', 'danger', 'info', 'success', 'warning';
+
+// The PascalCase form a color takes inside a class name, so `$prefix + color-suffix($color)` is
+// always the class a FluxColor prop maps to.
+@function color-suffix($color) {
+    @return string.to-upper-case(string.slice($color, 1, 1)) + string.slice($color, 2);
+}
+
+// Emits `.<prefix>Gray` through `.<prefix>Warning` and hands the palette name to the block, so a
+// component declares its color variants once instead of six times.
+//
```

Stylelint (17.14.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)